Disable multiarch support.

From: Tom Rothamel <tom@rothamel.us>

Since everything is a croos-compile, it's not needed.
---
 setup.py |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/setup.py b/setup.py
index c6023e1..dba2d0e 100644
--- a/setup.py
+++ b/setup.py
@@ -636,6 +636,8 @@ class PyBuildExt(build_ext):
     def add_multiarch_paths(self):
         # Debian/Ubuntu multiarch support.
         # https://wiki.ubuntu.com/MultiarchSpec
+        return
+    
         tmpfile = os.path.join(self.build_temp, 'multiarch')
         if not os.path.exists(self.build_temp):
             os.makedirs(self.build_temp)
